### ISSUE TITLE
Failed to render NunjucksModal

### ISSUE THREAD

**Issue by Haarolean** [getinsomnia/insomnia#2414]
**Describe the bug**
Clicking on previously added (long time ago) 'response -> body attribute' throws an exception. Works fine with recent placeholders. 

```
TypeError: Cannot read property 'value' of undefined
    at Object.hide (/Applications/Development/REST/Insomnia.app/Contents/Resources/app.asar/node_modules/insomnia-plugin-response/index.js:92:31)
    at TagEditor.renderArg (file:///Applications/Development/REST/Insomnia.app/Contents/Resources/app.asar/bundle.js:152425:67)
    at react__WEBPACK_IMPORTED_MODULE_0__.createElement.activeTagDefinition.args.map (file:///Applications/Development/REST/Insomnia.app/Contents/Resources/app.asar/bundle.js:152590:111)
    at Array.map (<anonymous>)
    at TagEditor.render (file:///Applications/Development/REST/Insomnia.app/Contents/Resources/app.asar/bundle.js:152590:76)
    at gi (file:///Applications/Development/REST/Insomnia.app/Contents/Resources/app.asar/bundle.js:174644:192)
    at fi (file:///Applications/Development/REST/Insomnia.app/Contents/Resources/app.asar/bundle.js:174643:224)
    at Rj (file:///Applications/Development/REST/Insomnia.app/Contents/Resources/app.asar/bundle.js:174725:490)
    at Qj (file:///Applications/Development/REST/Insomnia.app/Contents/Resources/app.asar/bundle.js:174708:199)
    at Kj (file:///Applications/Development/REST/Insomnia.app/Contents/Resources/app.asar/bundle.js:174708:128)

Filter
```

**To Reproduce**
no clue, can not reproduce for new placeholders, just the old ones.

**Expected behavior**
Modal window opens, I can edit the stuff.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS High Sierra
 - Installation Method: standalone
 - App Version 2020.3.3

**Additional context**

**Comment by thepisu** [comment]
I have trying redefining a parameter with different name, in the same environment, and exporting the environment.

Here we can see the difference, "token" is the old parameter (that caused error) , and "token2" is the new parameter (that works).

```
    "token":"{% response 'body', 'req_d8dffc4690bc450d9a7fe6ef47e1907d', '$.access_token' %}",
    "token2":"{% response 'body', 'req_d8dffc4690bc450d9a7fe6ef47e1907d', 'b64::JC5hY2Nlc3NfdG9rZW4=::46b', 'never', 60 %}",
```

Maybe this helps to diagnose the issue.

**Comment by develohpanda** [comment]
@thepisu thank you for that information, that definitely helped me to narrow down the cause 👍

**Comment by develohpanda** [comment]
Hmm, strange - it looks like #2643 _didn't_ fix this, at-least on Windows. Reopening it.

**Comment by develohpanda** [comment]
This has been released with [2020.4.1](https://github.com/Kong/insomnia/releases/tag/core%402020.4.1) 😄

**Comment by thepisu** [comment]
Thank you! I confirm that now it works.
